src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Empty",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1011"
+  },
+  "description": {
+    "text": "An empty project template for creating an ASP.NET Core application. This template does not have any content in it.",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1012"
+  },
+  "order": 100,
+  "icon": "vs-2017.3/Empty.png",
+  "learnMoreLink": "https://go.microsoft.com/fwlink/?LinkID=784883",
+  "uiFilters": [ "oneaspnet" ],
+  "supportsDocker": true,
+  "legacyTemplateIdentity": "Microsoft.NetCore.FSharp.EmptyWeb",
```

Comment:
@seancpeters Where can I confirm the "legacyTemplateIdentity" for all three F# templates?

Reply:
If this is not necessary I will remove it.  I do not think there were ever F# ASP.NET templates so there is no actual `legacyTemplateIdentity`.  What is this property used for?

Reply:
See my comment above in regards to legacyTemplateIdenity, you can leave it out for the F# templates.